stdlib/enum.pyi

```diff
@@ -58,6 +58,7 @@ class auto(IntFlag):
     def __new__(cls: Type[_T]) -> _T: ...
 
 class Flag(Enum):
+    name: Optional[int]  # type: ignore
```

Comment:
Maybe `str` instead of `int`?

```
>>> enum.Flag('Foo', ['A', 'B', 'C']).A.name
'A'
```

And a small nit: We prefer the new `Foo | None` syntax instead of `Optional[Foo], so you can do this:

```suggestion
    name: str | None  # type: ignore
```

Reply:
Yes, obviously the name should be a string, thanks for spotting that. I also updated the syntax.